src/languages/es.js

```diff
@@ -284,7 +284,9 @@ export default {
         imageUploadFailed: 'Error al cargar la imagen',
         deleteWorkspaceError: 'Lo sentimos, hubo un problema eliminando el avatar de su espacio de trabajo.',
         sizeExceeded: ({maxUploadSizeInMB}) => `La imagen supera el tamaño máximo de ${maxUploadSizeInMB}MB.`,
-        tooSmallResolution: ({minHeightInPx, minWidthInPx}) => `Por favor elige una imagen mas grande que ${minHeightInPx}x${minWidthInPx} píxeles`,
+        resolutionConstraints: ({
+            minHeightInPx, minWidthInPx, maxHeightInPx, maxWidthInPx,
+        }) => `Por favor elige una imagen mas grande que ${minHeightInPx}x${minWidthInPx} píxeles y mas pequena que ${maxHeightInPx}x${maxWidthInPx} píxeles`,
```

Comment:
I left the correct translation [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1672358985640889). For future reference, you should always get approval from an Expensify employee when it comes to translations to make sure it's correct and there are no grammar mistakes. We can all make mistakes for sure, but it is less likely to happen if you have it from us instead of someone else.

```suggestion
        }) => `Por favor, elige una imagen más grande que ${minHeightInPx}x${minWidthInPx} píxeles y más pequeña que ${maxHeightInPx}x${maxWidthInPx} píxeles`,
```

Reply:
Sorry, I thought that only `pixeles` part was changed

Reply:
updated